NalssiOtjang/Feature/RecordPhoto/Sources/RecordPhoto/CaptureCameraView/ViewModel/CaptureCameraViewModel.swift

```diff
@@ -61,6 +64,7 @@ final class CaptureCameraViewModel {
                     Log.error("CaptureCameraError")
```

Comment:
p3; 이 부분의 에러 내용을 상세하게 로깅하고. 캡쳐에 문제가 생겼을 때 사용자에게 피드백을 주면 좋을 듯?
```suggestion
                    Log.error("Failed to capture photo: \(error.localizedDescription)")
```

Reply:
확인